source/feature-documentation/gui/start.rst

```diff
@@ -17,4 +17,23 @@ By default Meshroom uses a unique id to name the output folders to prevent overw
 
 **Example**: You are not satisfied with your first result and make changes to the **StructureFromMotion** node. The new output will be placed under a different name inside the  **StructureFromMotion** Folder.
 
-You can change the **name of the output folders** of your nodes by clicking on the node and changing the **Output Folder** name in the **Attributes tab** of the **Graph Editor** Properties pane.
\ No newline at end of file
+You can change the **name of the output folders** of your nodes by clicking on the node and changing the **Output Folder** name in the **Attributes tab** of the **Graph Editor** Properties pane.
+
+External Reconstruction
```

Comment:
More details here https://github.com/alicevision/meshroom/wiki/Renderfarm-submitters
Some parts are drafts but OpenCue is a good start